src/bulkExport/updateStatus.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Handler } from 'aws-lambda';
+import AWS from 'aws-sdk';
+import { ExportJobStatus } from 'fhir-works-on-aws-interface';
+import DynamoDbParamBuilder from '../dataServices/dynamoDbParamBuilder';
+
+const EXPORT_JOB_STATUS = ['completed', 'failed', 'in-progress', 'canceled', 'canceling'];
+const isJobStatus = (x: string): x is ExportJobStatus => EXPORT_JOB_STATUS.includes(x);
+
+export const updateStatusStatusHandler: Handler<{ jobId: string; status: string }, void> = async event => {
+    const { jobId, status } = event;
```

Comment:
Need to validate jobId at all? Or is this Update also a create, or is that in the DDB parameter builder?

Reply:
It should be on the DDB param builder, but it is not. 
I just updated `DynamoDbParamBuilder` to update only existing jobs. In general We could use some refactoring around the ddb interactions. Relying on `DynamoDbParamBuilder` alone is not a great abstraction.